src/lib/Engineer.js

```diff
@@ -0,0 +1,20 @@
+const Employee = require("./Employee");
+
+//Add a class for the Engineer that inherits properties from the Employee Class
+class Engineer extends Employee {
```

Comment:
It would be preferable to prepare your classes to receive objects with keys of name, id, email and githuvUsername here rather than 4 separate parameters. This would look like:

```
constructor( {name, id, email, githubUsername } ) {
       super( {name, id, email} )
```

When creating an instance of the class, we would then need to also pass in an object.

The reason we would do this is to prevent any potential errors - when we have lots of arguments being passed into a Class/function, they have to be passed in the specific order that the class/function was prepped to receive them in. As we increase the number of arguments, the risk of accidentally passing them in in the wrong order increases, and this could cause bugs in our code.

By passing an object instead, we avoid this risk as we can pass an object with the relevant keys in any order. The destructure object we see in the function/class definition is just looking for an object containing those keys, it doesn't matter which order they come in.